app/components/Button/Button.css

```diff
@@ -1,52 +1,68 @@
 @import '~app/styles/variables.css';
 
 .button {
-  box-shadow: 1px 1px 3px 1px var(--color-gray-5);
-  font-weight: 600;
-  background: linear-gradient(
-    -180deg,
-    var(--color-red-3) 0%,
-    var(--color-red-2) 90%
-  );
-  padding: 10px 20px;
-  color: var(--color-light-gray-4);
-  outline: none;
-  border-radius: 4px;
+  display: inline-flex;
   align-items: center;
   justify-content: center;
-  display: inline-flex;
+  box-sizing: border-box;
+  height: 2.3em;
+  width: fit-content;
+  padding: 5px 15px;
+  box-shadow: 0 1px 0 rgba(27, 31, 36, 10%),
+    inset 0 1px 0 rgba(255, 255, 255, 3%);
+  font-weight: 600;
+  background-color: var(--color-light-gray-4);
+  color: var(--color-dark-gray-5);
+  outline: none;
+  border-radius: 6px;
+  border: 1px solid rgba(--color-dark-gray-4, 40%);
```

Comment:
Does this work without the `var()`??

Reply:
lol not at all, but for some reason it still worked last night 🙃 Nice catch!

Reply:
Perhaps it's better to just let the colors be hard coded instead ... 😞

Reply:
Hmm, yeah. I think we would need this to be stable in most browsers :/ https://www.w3.org/TR/css-color-5/#relative-colors
Sadly, no browsers have implemented it yet :cry: